roles/dnsmasq/templates/cifmw-dnsmasq.conf.j2

```diff
@@ -1,5 +1,5 @@
 # Managed by ci-framework/dnsmasq
-
+log-debug
```

Comment:
we may want to toggle this instead of having it on by default? You can easily pass it as a parameter in your env.
Or do you think debug logging all the time is really needed?

Reply:
Removed the log-debug.